### ISSUE TITLE
support for XMBC 13.2 Gotham?

### ISSUE THREAD

**Issue by stsichler** [generia/plugin.video.zdf_de_2016#1]
Is it possible to also get this working on XBMC 13.2 Gotham?
Thank you for your work.

**Comment by generia** [comment]
Just tested it. When lowering the xbmc.python depenency version to Gotham, the plugin is basically working (browsing ZDF.de, ...).
The only exception is that the video-player is not starting. Might be a small drawback.

I will have a look what is causing this. Stay tuned ...

**Comment by generia** [comment]
# Bad news - no HTTPS on Gotham
I looks like that the Gotham 13.2 player can not cope with https stream urls. ZDF.de is using https for streams. E.g. ZDF Live-TV:
https://zdf1314-lh.akamaihd.net/i/de14_v1@392878/master.m3u8

You can test this with the Simple IPTV Client. It shows the same problem. I tested with the following playlist:

#EXTM3U
#EXTINF:-1,Das Erste
http://live-lh.daserste.de/i/daserste_de@91204/index_184_av-p.m3u8?sd=10&rebase=on
#EXTINF:-1,ZDF
https://zdf1314-lh.akamaihd.net/i/de14_v1@392878/master.m3u8
#EXTINF:-1,ARD Live Stream local
http://localhost:8080/master-ard.m3u8
#EXTINF:-1,ZDF Live Stream local
http://localhost:8080/master-zdf-livestream.m3u8

When I proxy the streams to a http protocol using a local web-server the stream is shown by the player.

# Long story short
If you find a way to show https streams on Gotham, please let me know.

You can get the  zdf_de_2016 plugin running on Gotham by changing the fourth line in "[addon.xml](https://github.com/generia/plugin.video.zdf_de_2016/blob/master/addon.xml)" to 

`<import addon="xbmc.python" version="2.14.0"/>`

See also http://kodi.wiki/view/Addon.xml#Dependency_versions for details.

**Comment by stsichler** [comment]
Thank you for your investigations! Bad news, though.
I fear that it will force me to do a full system re-installation on my Odroid U3 in order to get Kodi running - a lot of work.

**Comment by stsichler** [comment]
I lowered the version requirement as you propsed to 2.14.0 and do you know what? **IT WORKS!!!**
Gotham *can* stream https, at least on my setup, so the https problems seem related to your test-setup.
Btw, this thread puzzled me: <https://ubuntuforums.org/showthread.php?t=1874780>
